src/kas/wallet/wallet.js

```diff
@@ -218,57 +225,111 @@ class Wallet {
         return this.apiInstances.registration
     }
 
-    // TODO: description, define return type, implementation
+    /**
+     * SinglePrivateKey presents a single private key such as account types of a legacy key or public key.
+     * @typedef {string} SinglePrivateKey
+     */
+
+    /**
+     * MutisigPrivateKeys presents a set of private keys of weighted multisig key account type.
+     * @typedef {string[]} MultisigPrivateKeys
+     */
+
+    /**
+     * RoleBasedPrivateKeys presents a set of private keys of role based account type. Additionally, each private key can contains multiple private keys
+     * for multisig account type.
+     * @typedef {MultisigPrivateKeys[]} RoleBasedPrivateKeys
+     */
+
+    /**
+     * MigrationAccount presents an account to be migrated. Each account can have one of three types of keys.
+     *
+     * @typedef {Object} MigrationAccount
+     * @property {string} address - Indicates Klaytn's address of an account.
+     * @property {number} nonce - Indicates nonce for a transaction
+     * @property {(SinglePrivateKey|MultisigPrivateKeys|RoleBasedPrivateKeys)} key - Indicates a set of private keys of an account.
+     */
+
     /**
      * Migrates Klaytn accounts to KAS Wallet API. <br>
-     * await caver.kas.wallet.migarateAccounts('./keystores', ['passphrase1', 'passphrase2', 'passphrase3']) <br>
-     * @param {string} dir The directory path to migrate accounts to KAS.
-     * @param {Array.<string>} passphrases The directory path to migrate accounts to KAS.
-     * @return {}
+     * await caver.kas.wallet.migarateAccounts([{address:'{{address}}', key:'{{private_keys}}'}]) <br>
+     *
+     * @param {number} chainId Chain ID.
+     * @param {MigrationAccount[]} accounts An array of account objects migrated into KAS.
+     * @return {StatusResponse}
      */
-    async migrateAccounts(dir, passphrases) {
+    async migrateAccounts(chainId, accounts) {
```

Comment:
Do we need to separately receive chainId as a parameter? If we can use `this.chainId` set by the user when initializing.

Reply:
I deleted `chainId` from the parameters.
I refactored the function to take only one parameter.